include/swift/AST/DiagnosticsSIL.def

```diff
@@ -71,7 +71,7 @@ ERROR(could_not_find_pointer_pointee_property,none,
 
 ERROR(writeback_overlap_property,none,
       "inout writeback to computed property %0 occurs in multiple arguments to"
-      " call, introducing invalid aliasing", (Identifier))
+      " call, introducing invalid aliasing", (DeclBaseName))
```

Comment:
This one's a property name, so it will never be a special name…but DeclBaseName is going to be more convenient from here on out. So the rule is "DeclBaseName unless you're actually talking about what the user wrote", I guess?